source/ui/MenuItem/index.jsx

```diff
@@ -34,11 +34,10 @@ const MenuItem = ({
       }}
     >
       {
-        icon
+        icon && icon
```

Comment:
I think this is redundant right?

Reply:
good point, yes this will just render undefined or null and be OK.